menu.html

```diff
@@ -131,5 +122,24 @@ <h2>New!</h2>
 </footer>
 </div>
 
+<script type="text/javascript" src="js/materialize.min.js"></script>
```

Comment:
This also is incorrect. Please see the documentation on the Getting Started page on the Materialize CSS website for the correct code for this.